src/main/scala/cromwell/engine/backend/CromwellBackend.scala

```diff
@@ -0,0 +1,41 @@
+package cromwell.engine.backend
+
+import akka.actor.ActorSystem
+import org.slf4j.LoggerFactory
+import Backend.BackendyString
+
+/**
+  * Provides a global singleton access to the instantiated backend. This isn't bulletproof and can lead to
+  * exceptions being thrown but the way the system utilizes it it shouldn't be a problem (famous last words).
+  * So far the only real oddities I've seen are from the unit tests when they were trying to go straight to the
+  * interior of the system.
+  *
+  * Also we expect nearly everything to change about Backend in the next couple of months due to Intel's
```

Comment:
Hey, you all are watching! :)

My original comment was a bit over the top. What I really meant was just that we expect to be moving towards a system where multiple backends are accessible from the same Cromwell so worrying too much about the best way to have our single system-wide backend accessible at the moment is not really worth it.

I do think it'd be good to go trolling through the weeds of your all's initial changes although based on office closures on both parts and the like early next year would probably be best.